src/main/webapp/WEB-INF/tags/instructor/moreInfo.tag

```diff
@@ -28,7 +28,6 @@
       <div class="panel-body">
         <span data-toggle="modal" data-target="#studentProfileMoreInfo"
             class="text-muted pull-right glyphicon glyphicon-resize-full cursor-pointer"></span>
-        <h5>More Info </h5>
```

Comment:
There are two places where the fix needs to be made. On clicking 
- `View`
- `All Records`

You have fixed the `View` part.

But on clicking `All Records` for a student, it now shows like this since you removed this.
![image](https://user-images.githubusercontent.com/12574756/36518846-2b22e83c-17c3-11e8-90f4-b54e0f5e4318.png)

Reply:
@GarrettCarrott for your reference, it's these 2 places under the `Students` page

![image](https://user-images.githubusercontent.com/26651556/36520756-e97f7610-17cd-11e8-8e1f-888de893a420.png)